test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -35,4 +36,7 @@ class AndroidRunCommand : Runnable {
 
     @Option(names = ["-h", "--help"], usageHelp = true, description = ["Prints this help message"])
     var usageHelpRequested: Boolean = false
+
+    @Option(names = ["--app"], description = ["The path to the application binary file."])
```

Comment:
we should borrow the descriptions from Firebase:

> The path to the application binary file. The path may be in the local filesystem or in Google Cloud Storage using gs:// notation.
>
> https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run

I think Flank supports `gs://` notation for remote apks.

Reply:
fixed